lib/scalingUtils.js

```diff
@@ -1,12 +1,18 @@
 import { Dimensions } from 'react-native';
 const { width, height } = Dimensions.get('window');
 
+//Guideline sizes are based on standard ~5" screen mobile device
 //Guideline sizes are based on standard ~5" screen mobile device
 const guidelineBaseWidth = 350;
 const guidelineBaseHeight = 680;
 
-const scale = size => width / guidelineBaseWidth * size;
-const verticalScale = size => height / guidelineBaseHeight * size;
-const moderateScale = (size, factor = 0.5) => size + ( scale(size) - size ) * factor;
+const widthRatio = width / guidelineBaseWidth;
+const heightRatio = height / guidelineBaseHeight;
+
+const scale = size => widthRatio * size;
+const verticalScale = size => heightRatio * size;
+const moderateScaleNormal = (size, factor = 0.5) => size + ( scale(size) - size ) * factor;
+const moderateScaleExtraSmall = (size, factor = 0.5) => width / moderateScaleNormal(size, factor) * width;
 
```

Comment:
Hey, sorry for the delay.
Not sure I understand the logic with the `moderateScaleExtraSmall` formula.
Could you explain it please? :)